### DISCUSSION TITLE
Wykorzystanie w Dashboard Energia

### DISCUSSION THREAD

**Discussion by krzyzak44** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#86] [Q&A]
Dzień dobry :)
jak Koledzy realizujecie u siebie wykorzystanie tej integracji w HA Dashboard Energia w kontekście jednodniowego przesunięcia danych?

**Comment by PiotrMachowski** [comment]
Od wersji v2.3.0 nie ma już przesunięcia w panelu Energia (ale tylko tam) ;)

**Reply by krzyzak44** [reply]
To przespałem :) i chyba coś źle robię bo aktualnie (pod datą 21.01) widzę konsumpcję z 20.01. Hmm...

**Reply by PiotrMachowski** [reply]
Bo zamiast encji do statystyk musisz dodać pozycje "tauron_importer". Wtedy będziesz miał opóźnienie w pojawianiu się danych, ale za to nie będzie w nich przesunięcia, będą dane dla każdej godziny, a na dodatek są zaciągnięte rok wstecz

**Reply by krzyzak44** [reply]
No faktycznie. Pięknie zadziałało! Podziękowania! (I kawa rzecz jasna ;))

**Reply by PiotrMachowski** [reply]
Dzięki!
